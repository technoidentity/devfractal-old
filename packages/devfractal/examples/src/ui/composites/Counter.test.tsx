import React from 'react'
import renderer, { ReactTestRenderer } from 'react-test-renderer'
import { Counter } from './Counter'

test('Counter', () => {
  const component: ReactTestRenderer = renderer.create(<Counter />)
  expect(component.toJSON()).toMatchSnapshot()
})
