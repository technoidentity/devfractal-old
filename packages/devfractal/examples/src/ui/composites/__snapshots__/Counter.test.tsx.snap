// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter 1`] = `
<div
  className="columns"
>
  <div
    className="column is-narrow"
  >
    <div
      className="box has-text-centered"
    >
      <button
        className="button is-info is-medium"
        onClick={[Function]}
        type="button"
      >
        <span
          className="icon"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-plus fa-w-14 "
            data-icon="plus"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 208H272V64c0-17.67-14.33-32-32-32h-32c-17.67 0-32 14.33-32 32v144H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h144v144c0 17.67 14.33 32 32 32h32c17.67 0 32-14.33 32-32V304h144c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </span>
      </button>
      <h1
        className="is-size-2"
      >
        0
      </h1>
      <button
        className="button is-success is-medium"
        onClick={[Function]}
        type="button"
      >
        <span
          className="icon"
        >
          <svg
            aria-hidden="true"
            className="svg-inline--fa fa-minus fa-w-14 "
            data-icon="minus"
            data-prefix="fas"
            focusable="false"
            role="img"
            style={Object {}}
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M416 208H32c-17.67 0-32 14.33-32 32v32c0 17.67 14.33 32 32 32h384c17.67 0 32-14.33 32-32v-32c0-17.67-14.33-32-32-32z"
              fill="currentColor"
              style={Object {}}
            />
          </svg>
        </span>
      </button>
    </div>
  </div>
  <div
    className="column has-text-centered"
  >
    <button
      className="button is-danger is-medium"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
